samcli/cli/context.py

```diff
@@ -50,9 +55,12 @@ def debug(self, value):
         self._debug = value
 
         if self._debug:
-            # Turn on debug logging
-            logging.getLogger("samcli").setLevel(logging.DEBUG)
-            logging.getLogger("aws_lambda_builders").setLevel(logging.DEBUG)
+            # Turn on debug logging and display timestamps
+            for logger_name in [SAM_CLI_LOGGER_NAME, LAMBDA_BULDERS_LOGGER_NAME]:
+                logger = logging.getLogger(logger_name)
+                logger.setLevel(logging.DEBUG)
+                for handler in logger.handlers:
```

Comment:
Why are we setting on all handlers?

Reply:
There is only one handlers I think. In order to get handler object from `logger`, only `logger.handlers` is available.

Reply:
(no longer applicable in the latest iteration)